### DISCUSSION TITLE
Audio over network

### DISCUSSION THREAD

**Discussion by Kooza25** [apohl79/audiogridder#648] [Q&A]
Hello there. Would it be possible route midi between 2 computers and receive audio channels from an audio interface connected to the remote computer?
I have synths connected to my studio system with Apollo card and I want to control them by midi and get audio back from Apollo ins, while controlling everything with a Macbook and the daw remotely

**Comment by apohl79** [comment]
AG does not support this. The server can only receive audio via hosted plugins, not from an audio interface. It can also not route midi to a midi device.
